
From: Geert Uytterhoeven <geert@linux-m68k.org>

M68k PCI: Removes unnecessary min/max macros and change calls to use kernel.h
macros instead.

Signed-off-by: Michael Veeck <michael.veeck@gmx.net>
Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/m68k/kernel/bios32.c |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff -puN arch/m68k/kernel/bios32.c~minmax-removal-arch-m68k-kernel-bios32c arch/m68k/kernel/bios32.c
--- 25/arch/m68k/kernel/bios32.c~minmax-removal-arch-m68k-kernel-bios32c	2004-10-02 18:40:53.032863608 -0700
+++ 25-akpm/arch/m68k/kernel/bios32.c	2004-10-02 18:40:53.037862848 -0700
@@ -46,8 +46,6 @@
 
 #define ALIGN(val,align)	(((val) + ((align) - 1)) & ~((align) - 1))
 
-#define MAX(val1, val2)		(((val1) > (val2)) ? val1 : val2)
-
 /*
  * Offsets relative to the I/O and memory base addresses from where resources
  * are allocated.
@@ -171,7 +169,7 @@ static void __init layout_dev(struct pci
 			 * Align to multiple of size of minimum base.
 			 */
 
-			alignto = MAX(0x040, size) ;
+			alignto = max_t(unsigned int, 0x040, size);
 			base = ALIGN(io_base, alignto);
 			io_base = base + size;
 			pci_write_config_dword(dev, reg, base | PCI_BASE_ADDRESS_SPACE_IO);
@@ -214,7 +212,7 @@ static void __init layout_dev(struct pci
 			 * Align to multiple of size of minimum base.
 			 */
 
-			alignto = MAX(0x1000, size) ;
+			alignto = max_t(unsigned int, 0x1000, size);
 			base = ALIGN(mem_base, alignto);
 			mem_base = base + size;
 			pci_write_config_dword(dev, reg, base);
_
